roles/ckan/templates/ckan/ckan_bootstrap.sh

```diff
@@ -1,9 +1,20 @@
 #!/bin/sh
 CONFIG=/etc/ckan/production.ini
+
+{% if ckan_datapusher_enable == 'true' %}
+echo "Making datastore user"
+psql "${CKAN_SQLALCHEMY_URL}" -c "CREATE ROLE datastore NOSUPERUSER NOCREATEDB NOCREATEROLE LOGIN PASSWORD '{{ ckan_ds_rw_pass }}';"
+echo "Making datastore_ro user"
+psql "${CKAN_SQLALCHEMY_URL}" -c "CREATE ROLE datastore_ro NOSUPERUSER NOCREATEDB NOCREATEROLE LOGIN PASSWORD '{{ ckan_ds_ro_pass }}';"
+echo "Making datastore db"
+psql "${CKAN_SQLALCHEMY_URL}" -c "CREATE DATABASE datastore OWNER ckan ENCODING 'utf-8';"
+echo "Setup datastore permissions"
+ckan -c "$CONFIG" datastore set-permissions | psql "${CKAN_SQLALCHEMY_URL}"
+{% endif %}
+
 # we're making sure to run our custom entrypoint.
 /usr/lib/ckan/ckan-entrypoint.sh
-echo "DB init:"
-ckan -c "$CONFIG" db init
```

Comment:
This is found inside the all project's ckan-entrypoints.  So is a doubling up of the db init script as far as I can tell and not needed.